### ISSUE TITLE
Replace `deps/domReady` with a wrapper for deprecation notice

### ISSUE THREAD

**Issue by saschanaz** [w3c/respec#1793]
Following #1767.

We may modify copydeps.js as:

```js
const srcDesMap = new Map([
  /* ... */
  ["./node_modules/domReady/domReady.js", "./js/deps/_domReady.js"],
  /* ... */
]);
```

and create a small wrapper for existing `deps/domReady` like this:

```js
import domReady from "deps/_domReady";
export default domReady;

// will run only when domReady is imported
console.warn("domReady dependency is deprecated in ReSpec.");
```

Thoughts? @marcoscaceres @sidvishnoi

**Comment by marcoscaceres** [comment]
Something like that would be nice.

**Comment by saschanaz** [comment]
Closed by #1797
